src/graphics/renderer.cpp

```diff
@@ -1,12 +1,21 @@
 #include "graphics/renderer.h"
-
 #include <sstream>
 
 namespace arch {
 
 void arch::OpenGLDebugMessagesHandler::init() {
     glEnable(GL_DEBUG_OUTPUT);
```

Comment:
Shouldn't glEnable also depend on this condition? In does not make sense to use it if we don't have access to debug message callback.

Reply:
done